common/HStream/Utils.hs

```diff
@@ -50,3 +53,9 @@ genUnique = do
        .|. fromIntegral (shiftL tsBit' 16)
        .|. fromIntegral rdmBit
 {-# INLINE genUnique #-}
+
+getCurrentTimestamp :: IO Int64
+getCurrentTimestamp = do
```

Comment:
How about `currentTimestampMS` to indicate this is a microsecond

Reply:
fixed